### DISCUSSION TITLE
selfhosted = privacy

### DISCUSSION THREAD

**Discussion by milahu** [pluja/awesome-privacy#277] [General]
i see very much overlap with
https://github.com/awesome-selfhosted/awesome-selfhosted

see also
https://www.inkandswitch.com/local-first/

usually privacy is a side-effect of selfhosting
because the best crypto is useless when your data is gone in a blackout

**Comment by pluja** [comment]
Yes, this is true. But not all self-hosted projects are privacy-focused. What do you suggest with this?

**Reply by milahu** [reply]
> What do you suggest

add a section "similar awesomes"?

**Comment by monokrome** [comment]
Privacy has nothing to do with being self-hosted, and privacy has nothing to do with whether your "data is gone in a blackout".

Consider, for instance, Plex. It doesn't seem to respect user privacy much and is self-hosted.

Adding to that, running something yourself in your own home without auditing or having done significant research to trust it could be exposing it to nearby services and data on your network, it can increase the surface area for attacks, and it could cause other privacy-adverse concerns.

Self-hosted != privacy in all cases, and conflating the two could be dangerous.
